plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -278,39 +275,12 @@ function parseIssueCover($node, $issue) {
 			if (is_a($n, 'DOMElement')) {
 				list($locale, $value) = $this->parseLocalizedContent($n);
 				switch ($n->tagName) {
-					case 'file_name': $issue->setFileName($value, $locale); break;
-					case 'original_file_name': $issue->setOriginalFileName($value, $locale); break;
-					case 'hide_cover_page_archives': $issue->setHideCoverPageArchives($value, $locale); break;
-					case 'hide_cover_page_cover': $issue->setHideCoverPageCover($value, $locale); break;
-					case 'show_cover_page': $issue->setShowCoverPage($value, $locale); break;
-					case 'cover_page_description': $issue->setCoverPageDescription($value, $locale); break;
-					case 'cover_page_alt_text': $issue->setCoverPageAltText($value, $locale); break;
-					case 'embed':
```

Comment:
could you also remove those elements from the plugins/importexport/native/native.xsd ?
also the entire pkp:issue_style element ?